### DISCUSSION TITLE
Hello, the new release doesn't seem to provide polygon-related APIs, please take a look

### DISCUSSION THREAD

**Discussion by whu-giser** [uber/h3-py#398] [Q&A]
1

**Comment by ajfriend** [comment]
Could you provide an example of what's missing and which version of h3-py you are using?

**Reply by whu-giser** [reply]
https://h3geo.org/docs/api/regions#polygontocells
I'm currently using '4.0.0b7', the API in the link above is missing, I looked at all the interfaces in the H3 library and there is no one
![1725536138390](https://github.com/user-attachments/assets/a8e5eca4-4363-4217-a226-fe778ece207c)

**Reply by ajfriend** [reply]
That website doesn't reflect the current `h3-py` v4 beta functions, which are listed here: https://uber.github.io/h3-py/api_quick.html#polygon-interface

We also have a tutorial here: https://uber.github.io/h3-py/polygon_tutorial.html

We should update that website when we do the final v4 `h3-py` release.
